libcxx/test/std/ranges/range.adaptors/range.drop/types.h

```diff
@@ -15,21 +15,21 @@
 int globalBuff[8];
 
 template <class T>
-struct sentinel {
+struct drop_sentinel {
```

Comment:
This change is unrelated to the commit in question. Is there a reason why it's being added here?

Reply:
This is needed because there is a `sentinel` type in the `test_range.h` file that causes a conflict when it is included to get the new `simple_view` concept. Do you have thoughts on how to handle it?

Reply:
Yes, but it's a whole-project change, so this is fine :)

Reply:
I completely understand!! Thanks!